dali/base/dasds.cpp

```diff
@@ -8882,6 +8797,138 @@ void CCovenSDSManager::notifyNode(CServerRemoteTree &node, PDState state)
 
 ///////////////////////
 
+bool CDeltaWriter::save(std::queue<Owned<CTransactionItem>> &todo)
+{
+    StringBuffer fname(dataPath);
+    fname.append(DELTAINPROGRESS);
+    OwnedIFile deltaIPIFile = createIFile(fname.append(DELTAINPROGRESS).str());
```

Comment:
did you mean to remove this append since you added one in the line above?

Reply:
whoops, well spotted. I'll revert it to how it was (remove line 8803)